src/Core/Content/LandingPage/SalesChannel/LandingPageRoute.php

```diff
@@ -68,12 +67,8 @@ public function load(string $landingPageId, Request $request, SalesChannelContex
             $resolverContext
         );
 
-        if (!$pages->has($pageId)) {
-            throw LandingPageException::notFound($pageId);
-        }
-
         $page = $pages->get($pageId);
-        if (!$page instanceof CmsPageEntity) {
+        if (!$page) {
```

Comment:
I would prefer an explicit check

Reply:
For PHPStan this is already CmsPageEntity|null so the null check should be fine, right?

Reply:
```suggestion
        if ($page === null) {
```

that is explicit :wink: 

`!` will match everything falsy, which could be tricky sometimes :grin:

Reply:
I don't think the CmsPageEntity could ever be falsy, but sure I'll change it

Reply:
I have updated the checks.